src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -1738,100 +1736,302 @@ <h2 class="text--center">Balance sheet</h2>
+                              {{if isNotEmpty $previous_total_assets}}
+                                   <ix:nonFraction contextRef="PY_END" decimals="0" unitRef="GBP" name="core:TotalAssetsLessCurrentLiabilities" format="ixt2:numdotdecimal">{{$previous_total_assets}}</ix:nonFraction>
+                              {{end}}
+                            </td>
+                        </tr>
+                  {{end}}
+
+              <!-- BALANCE SHEET - LIABILITIES - CREDITORS AFTER ONE YEAR -->
+
+                     {{if $creditors_after := $liabilities.creditors_amounts_falling_due_after_more_than_one_year}}
+                          <tr class="figures">
+                              <td id="creditors-after-label">Creditors: amounts falling due after more than one year:</td>
+
+                              <td class="figure noteIndex" id="uk-busCreditorsCreditorsAfterOneYear_CY_ENDNoteIndexes"></td>
+
+                              <td id="creditors-after-curr-val" class="strong figure">
+                                  {{if isNotEmpty $creditors_after.current_amount}}
+                                  {{if gt $creditors_after.current_amount 0.0}}({{end}}
+                                  <ix:nonFraction contextRef="CreditorsAfterOneYear_CY_END" decimals="0" unitRef="GBP" name="core:Creditors" format="ixt2:numdotdecimal">{{$creditors_after.current_amount}}</ix:nonFraction>
+                                  {{if gt $creditors_after.current_amount 0.0}}){{end}}
+                                  {{end}}
+                              </td>
+
+                              <td id="creditors-after-prev-val" class="figure">
+                                  {{if isNotEmpty $creditors_after.previous_amount}}
+                                  {{if gt $creditors_after.previous_amount 0.0}}({{end}}
+                                  <ix:nonFraction contextRef="CreditorsAfterOneYear_PY_END" decimals="0" unitRef="GBP" name="core:Creditors" format="ixt2:numdotdecimal">{{$creditors_after.previous_amount}}</ix:nonFraction>
+                                  {{if gt $creditors_after.previous_amount 0.0}}){{end}}
+                                  {{end}}
+                              </td>
+                          </tr>
+                      {{end}}
+
+              <!-- BALANCE SHEET - LIABILITIES - PROVISION FOR LIABILITIES -->
+
+                      {{if $provision := $liabilities.provision_for_liabilities}}
+                          <tr class="figures">
+                              <td id="provision-for-liabilities-label">Provision for liabilities:</td>
+
+                              <td class="figure noteIndex" id="uk-busProvisionsForLiabilitiesBalanceSheetSubtotalCY_ENDNoteIndexes"></td>
+
+                              <td id="provision-curr-val" class="strong figure">
+                                  {{if isNotEmpty $provision.current_amount}}
+                                  {{if gt $provision.current_amount 0.0}}({{end}}
```

Comment:
Do we use parentheses for positive amounts?

Reply:
These values are subtracted from the total, so any value over 0 is in brackets to show it is a deduction.

Reply:
Perfect. I couldn't remember the logic and there was nothing in the story.